config/management/src/genesis.rs

```diff
@@ -51,9 +52,9 @@ impl Genesis {
         Ok(genesis)
     }
 
-    /// Retrieves association key from the remote storage. Note, at this point in time, genesis
+    /// Retrieves the association key from the remote storage. Note, at this point in time, genesis
     /// only supports a single association key.
-    pub fn association(&self, layout: &Layout) -> Result<Ed25519PublicKey, Error> {
+    pub fn association_key(&self, layout: &Layout) -> Result<Ed25519PublicKey, Error> {
```

Comment:
Should we reconcile the terms `faucet_key`, `root_key`, and `association_key` into one consistent term?

Reply:
Agreed. As mentioned above, we'll delegate this to a follow up PR.